tests/unit/ViolationUtilsTest.ts

```diff
@@ -206,28 +211,38 @@ describe('getViolationsOnyxData', () => {
                         Accounting: {
                             name: 'Accounting',
                             enabled: true,
+                            // eslint-disable-next-line @typescript-eslint/naming-convention
+                            'GL Code': '',
                         },
                     },
                     required: true,
+                    orderWeight: 1,
                 },
                 Region: {
                     name: 'Region',
                     tags: {
                         Africa: {
                             name: 'Africa',
                             enabled: true,
+                            // eslint-disable-next-line @typescript-eslint/naming-convention
+                            'GL Code': '',
                         },
                     },
+                    required: true,
+                    orderWeight: 1,
                 },
                 Project: {
                     name: 'Project',
                     tags: {
                         Project1: {
                             name: 'Project1',
                             enabled: true,
+                            // eslint-disable-next-line @typescript-eslint/naming-convention
+                            'GL Code': '',
                         },
                     },
                     required: true,
+                    orderWeight: 1,
```

Comment:
Can you change the order and put Region after Project, but keep Region with orderWeight 2 and Project with orderWeight 3? The test should still pass, and I think it'll be a better test cause it'll be more likely that the "default" order we get when iterating over these is different than the one set by orderWeight
```suggestion
                    orderWeight: 3,
```